### PR TITLE
Add test_KNNClassificationVoter.py

### PR DESCRIPTION
**Pull request by williamrxu**
Issue #96, Sprint 0. Add unit tests for KNNClassificationVoter class.

### CODE REVIEW

File: proglearn/tests/test_KNNClassificationVoter.py [hunk 1]
```diff
@@ -9,7 +9,10 @@
     
 class TestKNNClassificationVoter(unittest.TestCase):
 
-    
+    def test_initialize(self):
+        KNNClassificationVoter(3)
+        self.assertTrue(True)
```
Comment (jdey4) [inline proglearn/tests/test_KNNClassificationVoter.py:14]: This line is wrong.

Comment (jdey4) [reply proglearn/tests/test_KNNClassificationVoter.py:14]: Is the initialization test necessary?

File: proglearn/tests/test_KNNClassificationVoter.py [hunk 2]
```diff
@@ -41,3 +44,5 @@ def test_correct_vote(self):
         assert_allclose(Y_test, kcv.vote(X_test), atol=1e-4)
```
Comment (jdey4) [inline proglearn/tests/test_KNNClassificationVoter.py:44]: X_test and Y_test do not have the same size.

**Review by jdey4 — CHANGES_REQUESTED** [review]

File: proglearn/tests/test_KNNClassificationVoter.py (hunk 2 above)
Comment (williamrxu) [reply proglearn/tests/test_KNNClassificationVoter.py:41]: They don't, but Y_test and kcv.vote(X_test) have the same size.

**Review by jdey4 — APPROVED** [review]

**Comment by jdey4** [comment]
@williamrxu could you solve the merge conflict?

**Comment by williamrxu** [comment]
I was not able to solve the merge conflict. I have deleted the branch, force reset my fork, made a new sprint0 branch, and created a new PR. New PR at Pull Request #165.

**Pull request closed without merging**